variables.tf

```diff
@@ -45,10 +45,16 @@ variable "helm_release_name" {
 }
 variable "helm_repo_url" {
   type        = string
-  default     = "https://charts.karpenter.sh"
+  default     = "public.ecr.aws"
   description = "Helm repository"
 }
 
+variable "helm_repo_oci" {
+  type        = bool
+  default     = false
```

Comment:
When `helm_repo_url` is by default defined as oci registry, we should also make `helm_repo_oci` enabled by default

Reply:
fixed